pkgs/development/python-modules/omemo/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+
+  setuptools,
+
+  typing-extensions,
+  xeddsa,
+
+  pytestCheckHook,
+  oldmemo,
+}:
+buildPythonPackage rec {
+  pname = "omemo";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Syndace";
+    repo = "python-omemo";
+    tag = "v${version}";
+    hash = "sha256-egb4UFoF/gS3LKutArnJSXxDYH/xyBLOxWec98rOT9Y=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    typing-extensions
+    xeddsa
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    oldmemo
+  ];
+
+  pythonImportsCheck = [
+    "omemo"
+  ];
+
+  passthru.tests = {
+    omemo.overridePythonAttrs = {
+      doCheck = true;
+    };
```

Comment:
```suggestion
    pytest = omemo.overridePythonAttrs {
      doCheck = true;
    };
```

Reply:
It should be also be `oldmemo`, not `omemo`

Reply:
No, omemo.

Reply:
Am I missing something, then?

```
       error: undefined variable 'omemo'
       at /home/user/nixpkgs/pkgs/development/python-modules/omemo/default.nix:47:14:
           46|   passthru.tests = {
           47|     pytest = omemo.overridePythonAttrs {
             |              ^
           48|       doCheck = true;
```

Reply:
oldmemo doesn't have any tests

Reply:
Oh, I get it, now. Thanks!